userguide/services.rst

```diff
@@ -1635,7 +1627,9 @@ unless there is a specific need.**
   back up the data, create a new case-insensitive dataset, create an
   SMB share on it, then copy the data from the old one onto it. After
   the data has been checked and verified on the new share, the old one
-  can be deleted.
+  can be deleted. Once the case-insensitive dataset is created, the
```

Comment:
My writing can also confuse native English speakers. :) The problem with the way that it is written originally is that no performance improvement will occur until the admin disables the case sensitivity checks. 

Another way of looking at it is that case sensitivity checks cause o(n^2) delay. Disabling them speeds things up. Making the dataset case insensitive allows you to do this safely. 

I'm open to better ways to say this.